Update two informtion_schema tables desc (#18801)
pingcap/docs: information-schema/information-schema-processlist.md

The `PROCESSLIST` table has additional columns not present in `SHOW PROCESSLIST`:

<CustomContent platform="tidb">

* A `DIGEST` column to show the digest of the SQL statement.
* A `MEM` column to show the memory used by the request that is being processed, in bytes.
* A `DISK` column to show the disk usage in bytes.
* A `TxnStart` column to show the start time of the transaction.
* A `RESOURCE_GROUP` column to show the resource group name.
* A `SESSION_ALIAS` column to show the alias of the current session.
* A `ROWS_AFFECTED` column to show the number of rows currently affected by the statement.

> * A `TIDB_CPU` column to show the CPU time (in nanoseconds) consumed by the TiDB server for the statement. This column shows meaningful data only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.

Reviewer: ```suggestion
* A `TIDB_CPU` column to show the amount of time in nanoseconds that the statement consumes the TiDB server CPU. This column shows meaningful value only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.
```

> * A `TIKV_CPU` column to show the CPU time (in nanoseconds) consumed by the TiKV server for the statement.

Reviewer: ```suggestion
* A `TIKV_CPU` column to show the amount of time in nanoseconds that the statement consumes the TiKV server CPU.
```

</CustomContent>

<CustomContent platform="tidb-cloud">

* A `DIGEST` column to show the digest of the SQL statement.
* A `MEM` column to show the memory used by the request that is being processed, in bytes.
* A `DISK` column to show the disk usage in bytes.
* A `TxnStart` column to show the start time of the transaction.
* A `RESOURCE_GROUP` column to show the resource group name.
* A `SESSION_ALIAS` column to show the alias of the current session.
* A `ROWS_AFFECTED` column to show the number of rows currently affected by the statement.

> * A `TIDB_CPU` column to show the CPU time (in nanoseconds) consumed by the TiDB server for the statement. This column shows meaningful data only when the [Top SQL](https://docs.pingcap.com/tidb/stable/top-sql) feature is enabled. Otherwise the value will be `0`.

Reviewer: ```suggestion
* A `TIDB_CPU` column to show the amount of time in nanoseconds that the statement consumes the TiDB server CPU. This column shows meaningful value only when the [Top SQL](https://docs.pingcap.com/tidb/stable/top-sql) feature is enabled. Otherwise the value will be `0`.
```

> * A `TIKV_CPU` column to show the CPU time (in nanoseconds) consumed by the TiKV server for the statement.

Reviewer: ```suggestion
* A `TIKV_CPU` column to show the amount of time in nanoseconds that the statement consumes the TiKV server CPU.
```

* A `RESOURCE_GROUP` column to show the resource group name.
* A `SESSION_ALIAS` column to show the alias of the current session.
* A `ROWS_AFFECTED` column to show the number of rows currently affected by the statement.

> * A `TIDB_CPU` column to show the `TIDB` server CPU time in seconds used by the statement.
> * A `TIKV_CPU` column to show the `TIKV` server CPU time in seconds used by the statement.

Reviewer 2: ```suggestion
* A `TIDB_CPU` column to display the CPU time (in seconds) consumed by the TiDB server for the statement.
* A `TIKV_CPU` column to display the CPU time (in seconds) consumed by the TiKV server for the statement.
```

> * A `TIDB_CPU` column to display the CPU time (in seconds) consumed by the TiDB server for the statement.
> * A `TIKV_CPU` column to display the CPU time (in seconds) consumed by the TiKV server for the statement.

Reviewer 2: ```suggestion
* A `TIDB_CPU` column to show the CPU time (in seconds) consumed by the TiDB server for the statement.
* A `TIKV_CPU` column to show the CPU time (in seconds) consumed by the TiKV server for the statement.
```

* `RESOURCE_GROUP`: The resource group name.
* `SESSION_ALIAS`: The alias of the current session.
* `ROWS_AFFECTED`: The number of rows currently affected by the statement.

> * `TIDB_CPU`: The CPU time (in nanoseconds) consumed by the TiDB server for the statement.

Reviewer 3: ```suggestion
* `TIDB_CPU`: The CPU time (in nanoseconds) consumed by the TiDB server for the statement. This column shows meaningful data only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.
```

> * `TIDB_CPU`: The CPU time (in nanoseconds) consumed by the TiDB server for the statement. This column shows meaningful data only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.

Reviewer: ```suggestion
* `TIDB_CPU`: The amount of time in nanoseconds that the statement consumes the TiDB server CPU. This column shows meaningful value only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.
```

> * `TIDB_CPU`: The amount of time in nanoseconds that the statement consumes the TiDB server CPU. This column shows meaningful value only when the [Top SQL](/dashboard/top-sql.md) feature is enabled. Otherwise the value will be `0`.

Reviewer: It seems that we also need to use <CustomContent platform="tidb-cloud"> for this. Because an error occurs in the ci check for cloud links.

Reviewer 2: Fixed

> * `TIKV_CPU`: The CPU time (in nanoseconds) consumed by the TiKV server for the statement.

Reviewer: ```suggestion
* `TIKV_CPU`: The amount of time in nanoseconds that the statement consumes the TiKV server CPU.
```
